clients/cli/src/main/java/org/apache/gravitino/cli/Properties.java

```diff
@@ -55,19 +55,21 @@ public Properties(String delimiter, String keyValueSeparator) {
    * <p>Each pair in the input string is split by the specified delimiter, and then each pair is
    * further split by the key-value separator.
    *
-   * @param input The input string containing name-value pairs.
+   * @param inputs An arrays of input strings containing name-value pairs.
    * @return A map of entries, where each entry represents a key-value pair from the input string.
    */
-  public Map<String, String> parse(String input) {
+  public Map<String, String> parse(String[] inputs) {
```

Comment:
Is this change related with the "user and group" CLI? seems it has already been handled in previous PR, please double check whether this is a code merge issue.

Reply:
It not a code merge issue - the way properties have been handled have been improved